docs/api.md

```diff
@@ -291,6 +292,14 @@ Passes the enable-es3-apis flag to the Chrome engine, to force the activation of
 ```
 Forces the packaged app to ignore web security errors, such as [Mixed Content](https://developer.mozilla.org/en-US/docs/Security/Mixed_content) errors when receiving HTTP content on a HTTPS site.
 
+
+#### [internal-urls]
+
+```
+--internal-urls <regex>
+```
+Customize what should open in an external browser. If the URL does not match the regex, it will open in an external browser.
```

Comment:
@cbess awwwesome, thanks for noticing this missing piece 👍. Users taught us `--internal-urls` has several interesting use cases, could you document them too using my comment at https://github.com/jiahaog/nativefier/issues/212#issuecomment-292689727 ? Feel free to clarify / reformulate as needed 🙂.

Reply:
Will do

Reply:
I think I made the changes you mentioned.